paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -701,7 +811,7 @@ def set_capacity(self, capacity):
                             "Not updating target capacity because this is a cancelled SFR, "
                             "we are just draining and killing the instances",
                         )
-                        return
+                        return None
```

Comment:
kinda hints that we should be raising something?

I'm surprised that this and line 822 are not bugs. 

Still ship though, it looks like it 'normally' returns a request response dict

Reply:
The returns here aren't actually checked outside of tests; I left them in to keep the test logic the same.